### PR TITLE
Creates side bar action

### PR DESCRIPTION
**Pull request by JuanequeX**
## Quick Info
This feature creates the side bar action which have  a list with taxonomies about products,
information about us and icons to social medias of sumus project.

## What does this change?
This change add a side bar action in the hamburgue icon in header.

## Why are you changing that?
Because need a side bar to show taxonomies of products and information about the sumus project.

## How do you manually test this?
Can test this feature navigate in the header of the project realize a click in the icon hamburguer.

## Screenshots (if apply)
* Can you see [HERE](https://drive.google.com/drive/folders/1wliaKNEGPtFYF8fvAcTaSIBCPdQ5xl6u?usp=sharing)

## Merge/Deploy Checklist
* [ ]   It has been reviewed and accepted.

### CODE REVIEW

File: app/javascript/components/header/Header.jsx [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+const Header = ({actionOpen, isOpen}) => {
+
   return (
     <header className='header'>
-      <a className='header__hamburger_icon'><img src={HamburgerLogo} alt="Menu" /></a>
+      <a className={isOpen ? 'header__hamburger-icon-close' :'header__hamburger_icon'}
+        onClick={actionOpen}>
+        <img onClick = {actionOpen}
+          src={isOpen ? LogoHeaderClose : HamburgerLogo} alt="Menu" />
+      </a>
```
Comment (OswaldoPineda) [inline app/javascript/components/header/Header.jsx:16]: change this tag, if you don't need an `href` you don't have to use the `a` tag

File: app/javascript/components/side-bar/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = () => {
+    fetch('/solidus/api/taxonomies.json', {
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer 49d9878bc37c2fb5c8e363224259f1a2f520f57cebf69c3e'
```
Comment (OswaldoPineda) [inline app/javascript/components/side-bar/Sidebar.jsx:21]: change this to use ENV vars

File: app/javascript/components/side-bar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      .then((data) => {
+        setTaxonomies(data.taxonomies);
+    });
+  }
+
+    return (
+    <>
+      <div onClick={actionClose} data-show={isOpen} className='shadow-background'></div>
+      <div className="sidebar">
+        <div className = {`side-bar-master ${ isOpen ? 'side-bar-master-show' : 'side-bar-master-hidden' }`} >
```
Comment (OswaldoPineda) [inline app/javascript/components/side-bar/Sidebar.jsx:34]: remove the spaces around the `=`

File: app/javascript/components/side-bar/sidebar.scss [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+  &[data-show="true"] {
+    display: block;
+    transition: all 0.5s ease-out;
+  }
+}
+
+.sidebar {
+  .side-bar-master-hidden {
+    left: -100%;
```
Comment (OswaldoPineda) [inline app/javascript/components/side-bar/sidebar.scss:25]: refactor this

**Review by OswaldoPineda — CHANGES_REQUESTED** [review]
just some changes

File: app/javascript/components/header/Header.jsx (hunk 1 above)
Comment (JuanequeX) [reply app/javascript/components/header/Header.jsx:16]: i will now using span for this

File: app/javascript/components/side-bar/Sidebar.jsx (hunk 3 above)
Comment (JuanequeX) [reply app/javascript/components/side-bar/Sidebar.jsx:34]: ok

File: app/javascript/components/side-bar/sidebar.scss (hunk 4 above)
Comment (JuanequeX) [reply app/javascript/components/side-bar/sidebar.scss:25]: Ready

File: app/javascript/components/side-bar/Sidebar.jsx (hunk 2 above)
Comment (JuanequeX) [reply app/javascript/components/side-bar/Sidebar.jsx:21]: I solution this extracting the value front solidus and rendering it in index.html.erb

**Review by OswaldoPineda — APPROVED** [review]

**Pull request merged**